### ISSUE TITLE
Scanner not recognizing headered NES roms

### ISSUE THREAD

**Issue by slanderous-mambo** [libretro/ludo#461]
When I try to scan NES roms, Ludo finds nothing. Scanning works if I cut off the first 16 bytes, since these are proper No-Intro files:
```
$ tail -c +17 Mega\ Man\ 2\ \(USA\).nes | sha1sum
2ec08f9341003ded125458df8697ca5ef09d2209  -
```
As of 1b30243c647245d90d12e4a073d87a7f904994ae, there is header skipping logic in the scanner, so this should work.

**Comment by kivutar** [comment]
It's weird because my headered ROMs are still scanning fine:

<img width="1072" alt="Screen Shot 2021-09-02 at 19 37 23" src="https://user-images.githubusercontent.com/442722/131844663-97a3300a-06ed-46fa-9596-c772e5e02e0f.png">

Are your ROMs zipped? Mine are and maybe I implemented the code wrong, only for zipped files?

**Comment by slanderous-mambo** [comment]
You are right! My ROMs are not zipped and zipping them makes the scanner work. I suppose I could have realized that myself, if I had read [the comment on top](https://github.com/libretro/ludo/blob/b83eff436906911b80d17c33bc736846194fe0e8/scanner/scanner.go#L2).

After reading the code, this definitely makes sense: the header only gets cut off [for zips](https://github.com/libretro/ludo/blob/b83eff436906911b80d17c33bc736846194fe0e8/scanner/scanner.go#L108-L133), and not [for .nes files](https://github.com/libretro/ludo/blob/b83eff436906911b80d17c33bc736846194fe0e8/scanner/scanner.go#L138-L146).
